frontend/public/components/modals/delete-namespace-modal.tsx

```diff
@@ -46,29 +48,36 @@ export const DeleteNamespaceModal: React.FC<DeleteNamespaceModalProps> = ({
   return (
     <form onSubmit={onSubmit} name="form" className="modal-content ">
       <ModalTitle className="modal-header">
-        <YellowExclamationTriangleIcon className="co-icon-space-r" /> Delete {kind.label}?
+        <YellowExclamationTriangleIcon className="co-icon-space-r" />{' '}
+        {t('public~Delete {{label}}?', { label: kind.label })}
       </ModalTitle>
       <ModalBody>
         <p>
-          This action cannot be undone. It will destroy all pods, services and other objects in the
-          namespace <strong className="co-break-word">{resource.metadata.name}</strong>.
+          {t(
+            'public~This action cannot be undone. It will destroy all pods, services and other objects in the namespace',
+          )}{' '}
+          <strong className="co-break-word">{resource.metadata.name}</strong>.
         </p>
         <p>
-          Confirm deletion by typing{' '}
-          <strong className="co-break-word">{resource.metadata.name}</strong> below:
+          <Trans ns="public">
+            Confirm deletion by typing{' '}
+            <strong className="co-break-word">{{ name: resource.metadata.name }}</strong> below:
+          </Trans>
         </p>
         <input
           type="text"
           data-test="project-name-input"
           className="pf-c-form-control"
           onKeyUp={onKeyUp}
-          placeholder="Enter name"
-          aria-label={`Enter the name of the ${kind.label} to delete`}
+          placeholder={t('public~Enter name')}
+          aria-label={t('public~Enter the name of the {{label}} to delete}', {
```

Comment:
We should hard-code the kind here, or if that's not possible, use kind.labelKey.

Reply:
Switched to `kind.labelKey`